extras/bin/ideascube

```diff
@@ -17,4 +17,11 @@ export DEBUG=$DEBUG
 export STORAGE_ROOT=$STORAGE_ROOT
 export IDEASCUBE_ID=$IDEASCUBE_ID
 
+
+if [ "$1" == "start" ]; then
```

Comment:
Not sure what is the best entry point, then: wsgi.py will not be loaded when running locally with manage.py, and vice versa.
Maybe hooking in wsgi.py is better, guessing that in dev process the migrate command will be run by hand, and the collectstatic is useless.